src/site/styles/layout.less

```diff
@@ -93,11 +93,46 @@ input {
     position: relative;
 }
 
-.action-confirmation {
-    position: absolute;
-    top: 0;
-    left: 0;
-    background: white;
+.action-confirmation { 
+    .action-confirmation-overlay {
+        z-index: 200;
+        background: black;
+        position: fixed;
+        opacity: .5;
+        width: 100%;
+        height: 100%;
+        bottom: 0;
+        left: 0;
+        top: 0;
+        right: 0;
+    }
+
+    .action-confirmation-container {
```

Comment:
I'm basically saying to not try and make everything look perfect at once. Get the functionality in and separate the fit and finish pieces for another PR.